airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -2004,6 +2004,11 @@ definitions:
         title: Request Option
         description: A request option describing where the parent key value should be injected into and under what field name if applicable.
         "$ref": "#/definitions/RequestOption"
+      incremental_dependency:
```

Comment:
I'm trying to think if there's a more descriptive word for this field. `incremental_dependency` doesn't provide a clear meaning to what this is doing.

Maybe something like `incremental_sync_based_on_child`? I'm not attached to it and the naming is not a blocker, but trying to consider how we can make it more clear what this boolean does.